.github/workflows/ci.yaml

```diff
@@ -20,16 +20,14 @@ jobs:
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v2
-      - name: Mount bazel action cache
+      - name: Mount bazel caches
         uses: actions/cache@v2
         with:
-          path: "~/.cache/bazel"
-          key: bazel
-      - name: Mount bazel repo cache
-        uses: actions/cache@v2
-        with:
-          path: "~/.cache/bazel-repo"
-          key: bazel-repo
+          path: |
+            "~/.cache/bazel"
+            "~/.cache/bazel-repo"
+          key: bazel-cache-${{ hashFiles('**/BUILD.bazel', '**/*.bzl', 'WORKSPACE') }}
```

Comment:
hmm. why the cache key? are we concerned about hitting the max size for the underlying storage? where are these caches stored actually?

Reply:
Some context here: https://github.com/bazel-contrib/rules-template/pull/4/files.